README.md

```diff
@@ -37,27 +50,31 @@ We test this foreign data wrapper on Ubuntu Server 18.04 using PostgreSQL-11 tog
   sudo ldconfig
   ```
 
-- Build this foreign data wrapper with RocksDB:
+- Build this foreign data wrapper:
   
   ```sh
+  git clone https://github.com/vidardb/PostgresForeignDataWrapper.git
+
   cd PostgresForeignDataWrapper 
 
   make
 
   sudo make install
   ```
 
-- Build this foreign data wrapper with VidarDB:
+  If you want to build this foreign data wrapper for [VidarDB](https://github.com/vidardb/vidardb), you can build like this:
```

Comment:
Thanks for making this change! But it is still not clear to me when to use `make VIDARDB=true`.
Maybe say something like:

To build the foreign data wrapper for VidarDB, add flag `VIDARDB=true` to the above `make` command.